gr-digital/lib/crc16_async_bb_impl.h

```diff
@@ -0,0 +1,48 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2021 Cesar Martinez.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ * 
+ */
+
+#ifndef INCLUDED_DIGITAL_CRC16_ASYNC_BB_IMPL_H
+#define INCLUDED_DIGITAL_CRC16_ASYNC_BB_IMPL_H
+
+#include <gnuradio/digital/crc16_async_bb.h>
+#include <boost/crc.hpp>
+
+namespace gr {
+namespace digital {
+
+class crc16_async_bb_impl : public crc16_async_bb
+{
+private:
+    
+    boost::crc_optimal<16, 0x1021, 0xFFFF, 0, false, false> d_crc_ccitt_impl;
```

Comment:
Here you can use `boost::crc_ccitt_type` instead of `boost::crc_optimal<16, 0x1021, 0xFFFF, 0, false, false>`, by the way.